### PR TITLE
Fixed issue for T5 base model quantization issue with IPEX smoothquant

### PR DESCRIPTION
**Pull request by PenghuiCheng**
## Type of Change

bug fix 
No API changed

## Description

Fixed quantization failure when T5 base model be quantized with IPEX smoothquant.

## Expected Behavior & Potential Risk

Quantize successfully

## How has this PR been tested?

Local tested

### CODE REVIEW

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/ITREX-release/job/ITREX-1.3-release-test/35/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/ITREX-release/job/ITREX-1.3-release-test/37

**Comment by VincyZhang** [comment]
@PenghuiCheng  please check!
Traceback (most recent call last):
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/quantization.py", line 234, in fit
    strategy.traverse()
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/strategy/auto.py", line 140, in traverse
    super().traverse()
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/strategy/strategy.py", line 482, in traverse
    self._setup_pre_tuning_algo_scheduler()
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/strategy/strategy.py", line 360, in _setup_pre_tuning_algo_scheduler
    self.model = self._pre_tuning_algo_scheduler("pre_quantization")
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/algorithm/algorithm.py", line 127, in __call__
    self._q_model = algo(self._origin_model, self._q_model, self._adaptor, self._dataloader, self._calib_iter)
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/algorithm/smooth_quant.py", line 89, in __call__
    q_model = adaptor.smooth_quant(
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/adaptor/pytorch.py", line 1796, in smooth_quant
    model._model = self.sq.transform(
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/adaptor/torch_utils/smooth_quant.py", line 1286, in transform
    out_pre_sq = model_forward_per_sample(self.model, example_inputs, self.device)
  File "/home/sdp/miniconda3/envs/pt-ipex-2.1.0+cpu-3.10-spr/lib/python3.10/site-packages/neural_compressor/adaptor/torch_utils/smooth_quant.py", line 110, in model_forward_per_sample
    output = forward_wrapper(model, sample[0], device)
KeyError: 0

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/ITREX-release/job/ITREX-1.3-release-test/40/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/ITREX-release/job/ITREX-1.3-release-test/50/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/ITREX-release/job/ITREX-1.3-release-test/51/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/3482/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/3500/

File: requirements.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ cmake>=3.16
 py-cpuinfo
 setuptools>=65
 setuptools_scm[toml]>=6.2
+wheel
```
Comment (hshen14) [inline requirements.txt:6]: is it a real package?

**Comment by VincyZhang** [comment]
> https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/3500/

@PenghuiCheng please check the error, your json format is not correct

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/3547/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/3556/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/3557/

**Comment by VincyZhang** [comment]
@PenghuiCheng Tuning and benchmark are ok, but acc can not run. Please fix in next PR.

**Pull request merged**